docs/source/index.rst

```diff
@@ -1,6 +1,15 @@
 Welcome to developer documentation for Leapp!
-=================================
-   `Leapp <https://oamg.github.io/leapp/>`_ is an OS and application modernization framework
+=============================================
+`Leapp <https://oamg.github.io/leapp/>`_ is an OS and application modernization framework.
+
+Resources not included in this RTD:
+ - RPM dependencies for `leapp-repository <https://github.com/oamg/leapp-repository/blob/master/docs/dependencies.md>`_.
+ - Upgrade inhibition by producing a `inhibiting report message <https://github.com/oamg/leapp-repository/blob/master/docs/howto-inhibit-upgrade.md>`_.
+ - Type of tests used for `testing the actors <https://github.com/oamg/leapp-repository/blob/master/docs/tests.md>`_.
+ - NVR schema used for for leapp-related `rpm packages <https://github.com/oamg/leapp-guidelines/blob/master/build-schema.md>`_.
+ - Overview (slightly outdated) of phases used in `Inplace upgrades Workflow <https://raw.githubusercontent.com/leapp-to/leapp-to.github.io/master/img/phases.png>`_.
+   For up-to-date state, consult the `source of IPU workflow <https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/workflows/inplace_upgrade.py>`_.
+ - Architecture overview of `leapp and snactor tools <https://github.com/leapp-to/leapp-to.github.io/blob/master/architecture.md>`_.
```

Comment:
This link points to the archived repo. I think we should move the content directly to the RTD.